modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/portlet/ContentDashboardAdminPortlet.java

```diff
@@ -142,13 +173,36 @@ protected void deactivate() {
 		_contentDashboardConfiguration = null;
 	}
 
+	private List<AssetVocabulary> _getAssetVocabularies(long groupId) {
```

Comment:
Better if we move this code outside of the portlet, like an AssetVocabularies Util or something like this, you can notice that we don't have any logic in the Portlet because it is really hard to test something here :(.